docs/src/terminology.md

```diff
@@ -50,7 +50,7 @@ The wallclock duration between a [leader](terminology.md#leader) creating a [tic
 
 ## confirmed block
 
-A [block](terminology.md#block) that has received a [supermajority](terminology.md#supermajority) of [ledger votes](terminology.md#ledger-vote) with a ledger interpretation that matches the leader's.
+A grouping of transactions and other data ([block](terminology.md#block)) that has received a [supermajority](terminology.md#supermajority) of [ledger votes](terminology.md#ledger-vote) that matches the leader's.
```

Comment:
this is probably unnecessary
`grouping of transactions and other data ([block](terminology.md#block))`

Reply:
```
A [block](terminology.md#block) that has received a [supermajority](terminology.md#supermajority) of [ledger votes](terminology.md#ledger-vote).
```

Reply:
tx...will change 'grouping' to 'record'.    any ideas on how to have a better non-circular definition?

Reply:
Hmm, I was thinking the link would suffice here...

Reply:
yeah this may have been a little overkill...i'll think about this